Templates/CppToolGem/Template/Code/CMakeLists.txt

```diff
@@ -101,15 +101,17 @@ if(PAL_TRAIT_BUILD_HOST_TOOLS)
     )
 
     # The ${Name}.Editor.Private.Object target is an internal target
-    # which is only to be used by this Gem's CMakeLists.txt and any Subdirectories
-    # Other Gems should not use this target
+    # which is only to be used by this gems CMakeLists.txt and any subdirectories
+    # Other gems should not use this target
```

Comment:
All of them have been changed to STATIC except for this one, which is still OBJECT, is that intentional?

Reply:
No it is not intentional. This should be updated as well